.github/workflows/yamllint.yml

```diff
@@ -0,0 +1,49 @@
+# yamllint disable rule:line-length rule:document-start rule:truthy
+name: PR yamllint check
+on: pull_request
+
+# This action only need a single permission in order to autoformat the code.
```

Comment:
The action doesn't autoformat, so either the comment is wrong, or the code is.

Reply:
Happens when you yy a file to not start from scratch 😅